modules/spacelift/README.md

```diff
@@ -97,20 +97,23 @@ chmod +x rootfs/usr/local/bin/spacelift*
 **NOTE**: this won't be necessary for Kubernetes implementations once Spacelift adds support for Kubernetes worker pools.
 
 1. Clone the following GitHub repository: `git clone git@github.com:spacelift-io/spacelift-worker-image.git`
-2. In the cloned repo, apply the [contrib/spacelift.pkr.hcl.patch](contrib/spacelift.pkr.hcl.patch) patch
-
-   ```bash
-   git apply spacelift.pkr.hcl.patch
-   ```
-
-   Change the region as needed, but make sure you update the `base_ami` with the latest Amazon Linux 2 AMI id:
-
-   ```bash
-   aws ec2 describe-images --owners amazon --filters "Name=name,Values=amzn2-ami-hvm-2.0.202*-x86_64-gp2" --query 'sort_by(Images, &CreationDate)[-1]'
-   ```
-
-3. Run `packer build spacelift.pkr.hcl` against the account Spacelift will be deployed to
-4. Take note of the AMI created and use it to configure the `spacelift-worker-pool` component
+1. Grab the latest AL2 AMI ID:
```

Comment:
The filter already takes care of this so no more need to pass in the hard coded ami.

Reply:
See this page https://github.com/spacelift-io/spacelift-worker-image/blob/main/aws/README.md

Packer isn't even needed anymore using the data source in the readme since spacelift packs images for you in desired regions. They are also open to packing their images in mew regions.